From 862094ec9897a8d7fca90b93582f886a770926b1 Mon Sep 17 00:00:00 2001
From: Hsun Lai <i@chainsx.cn>
Date: Wed, 21 May 2025 21:11:06 +0800
Subject: [PATCH 12/47] dt-bindings: vendor-prefixes: Add SakuraPi prefix

Add vendor prefix for SakuraPi.org, which produces
development boards like the SakuraPi-RK3308B.

Link: https://docs.sakurapi.org

Signed-off-by: Hsun Lai <i@chainsx.cn>
Acked-by: Krzysztof Kozlowski <krzysztof.kozlowski@linaro.org>
---
 Documentation/devicetree/bindings/vendor-prefixes.yaml | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/Documentation/devicetree/bindings/vendor-prefixes.yaml b/Documentation/devicetree/bindings/vendor-prefixes.yaml
index 7a264a372..fc345ff85 100644
--- a/Documentation/devicetree/bindings/vendor-prefixes.yaml
+++ b/Documentation/devicetree/bindings/vendor-prefixes.yaml
@@ -1304,6 +1304,8 @@ patternProperties:
     description: Recharge Véhicule Électrique (RVE) inc.
   "^saef,.*":
     description: Saef Technology Limited
+  "^sakurapi,.*":
+    description: SakuraPi.org
   "^samsung,.*":
     description: Samsung Semiconductor
   "^samtec,.*":
-- 
2.39.5

